### PR TITLE
chore(#27): Add dbt unit test brac

### PR DESCRIPTION
**Pull request by lorerod**
Add unit test to BRAC dbt models

closes https://github.com/medic/cht-pipeline/issues/27

### CODE REVIEW

**Comment by lorerod** [comment]
Hi reviewers! I implemented a pretty straightforward test on each of the BRAC models. If you have time, you can learn about dbt unit test [here](https://docs.getdbt.com/docs/build/tests). 
I also made a summary in [this](https://drive.google.com/file/d/1GuurvuS3CGp4wy9vYo1r8lGcIMa_binX/view) short video.
I hope to find a way to calculate test coverage. I will let you know if I do.
Thanks!

**Review by andrablaj — APPROVED** [review]
I managed to run the unit tests successfully. Nevertheless, I didn't check the test coverage resources you shared. I think this is already a good start for learning how to set up the unit tests for `dbt`. 

Screenshot 2023-10-26 at 13 48 40

File: tests/run_dbt_tests.sh [hunk 1]
```diff
@@ -17,4 +17,7 @@ echo Running dbt ...
 dbt run
 echo Running tests ...
 dbt test
+#echo Generating documentation...
```
Comment (andrablaj) [inline tests/run_dbt_tests.sh:20]: Are these lines needed, as they seem commented?

Comment (lorerod) [reply tests/run_dbt_tests.sh:20]: Deleted! thanks

**Comment by lorerod** [comment]
@andrablaj I will configure the project to check for test coverage on [this](https://github.com/medic/cht-pipeline/issues/34) issue.

**Pull request merged**